### PR TITLE
slight improvements for randomised names

### PR DESCRIPTION
**Pull request by MaxIsJoe**
CL: [New] Cyborgs will now have their names randomized.
CL: [New] Ghost Roles will now spawn you with randomly generated character sheets.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRole.cs [hunk 1]
```diff
@@ -153,6 +155,7 @@ private void EnableDefaultRespawning()
 		private void SpawnPlayer(PlayerInfo player)
 		{
 			playersSpawned++;
+			player.Mind.CurrentCharacterSettings = CharacterSheet.GenerateRandomCharacter();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRole.cs]: Maybe should be a setting on the ghost role itself, iif it Re-roles the character?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRole.cs]: there's no case where you want ghost roles to carry the same name currently.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRole.cs]: I think it's worth adding

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Systems/GhostRoles/GhostRole.cs]: done

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**